build.sbt

```diff
@@ -8,28 +8,32 @@ ThisBuild / organizationHomepage := Some(url("https://aiven.io/"))
 
 ThisBuild / resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
 
-val akkaVersion                = "2.6.20"
-val akkaHttpVersion            = "10.2.10"
-val alpakkaKafkaVersion        = "3.0.1"
-val kafkaClientsVersion        = "3.4.0"
-val alpakkaVersion             = "4.0.0"
-val futilesVersion             = "2.0.2"
-val quillJdbcMonixVersion      = "3.7.2"
-val postgresqlJdbcVersion      = "42.6.0"
-val scalaLoggingVersion        = "3.9.5"
-val logbackClassicVersion      = "1.4.7"
-val declineVersion             = "2.4.1"
-val pureConfigVersion          = "0.17.4"
-val scalaTestVersion           = "3.2.16"
-val scalaTestScalaCheckVersion = s"$scalaTestVersion.0"
-val akkaStreamsJson            = "0.9.0"
-val diffxVersion               = "0.8.3"
-val testContainersVersion      = "0.40.16"
-val testContainersJavaVersion  = "1.18.3"
-val scalaCheckVersion          = "1.17.0"
-val scalaCheckOpsVersion       = "2.10.0"
-val enumeratumVersion          = "1.7.2"
-val organizeImportsVersion     = "0.6.0"
+// TODO: Remove when Pekko has a proper release
+ThisBuild / resolvers += Resolver.ApacheMavenSnapshotsRepo
+ThisBuild / updateOptions := updateOptions.value.withLatestSnapshots(false)
```

Comment:
This is necessary because otherwise sbt will do multiple requests for the exact same snapshot version. Since the snapshots for Pekko use the git tag in the version, they won't ever get updated so its fine to cache the artifacts (in fact we kind of need to do this because we are currently overloading the Apache snapshots repos a bit).